sources/Outputs/include/Par.h

```diff
@@ -42,14 +42,17 @@ class Par {
      *
      * @param base basename for current simulation
      * @param filename file path for output PAR file (corresponds to basename)
+     * @param dirname the dirname of the output PAR file
```

Comment:
Wrong parameter name and wrong parameter order (name 'dir' below)

Reply:
OK